lisp/Makefile

```diff
@@ -38,7 +38,7 @@ install: dist
 	$(INSTALL) $(ELC) $(LISPDIR)
 	if [ -f /etc/debian_version -a -n "$(SITELISP)" -a ! -f "$(SITELISP)/ess-site.el" ] ; \
 	then \
-		ln -s $(LISPDIR)/ess-site.el $(SITELISP)/ess-site.el ; \
+		cd $(SITELISP) ; ln -s ess/ess-site.el . ; \
```

Comment:
Why this change?

Reply:
This actually came out of the rpm testing.
`buildrpm` (correctly!) complained about the symlink being absolute.  It makes a lot of sense that symlinks in the tarball "microcosm" ought to be relative to avoid assuming anything about where the files ultimately end up.